composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "candasm/yii1-mobile-detect-component",
+  "version": "1.0.1",
+  "type": "yii-extension",
+  "description": "MobileDetect library for YiiFramework 1.x",
+  "authors": [
+    {
+      "name": "Candasm",
+      "email": "candas@minareci.net"
+    }
+  ],
+  "config": {
+    "vendor-dir": "protected/vendor/"
+  },
+  "autoload": {
+    "psr-4": {
+      "": "src/"
+    }
+  },
+  "homepage": "https://github.com/candasm/MobileDetect",
+  "require": {
+    "yiisoft/yii": "1.1.14",
+    "mobiledetect/mobiledetectlib": "~2.7.6"
```

Comment:
Hard fixation version dependencies may create a conflict if a project that uses the package will have a dependence, which also requires MobileDetect, but another version. If possible, it is desirable to use *